pipelines/examples/clusters/sandbox.yaml

```diff
@@ -0,0 +1,9 @@
+account_name: x
+splunk_hec_token: x \
```

Comment:
Most of this file doesn't look like valid yaml...

Reply:
Yes, this file does not do anything yet... however I shall fix it